deps.windows/50-swig.ps1

```diff
@@ -90,20 +91,19 @@ function Fixup {
     $Items = @(
         @{
             Path = "$($ConfigData.OutputPath)/swig/bin/swig.exe"
-            Destination = "$($ConfigData.OutputPath)/swig/swig.exe"
+            Destination = "$($ConfigData.OutputPath)/bin/swig.exe"
         },
         @{
             Path = "$($ConfigData.OutputPath)/swig/share/swig/${Version}"
-            Destination = "$($ConfigData.OutputPath)/swig/Lib"
+            Destination = "$($ConfigData.OutputPath)/bin/Lib"
         }
     )
 
     $Items | ForEach-Object {
         $Item = $_
         Log-Status ('{0} => {1}' -f $Item.Path, $Item.Destination)
-        Move-Item @Item
+        Move-Item @Item -Force
     }
 
-    Remove-Item "$($ConfigData.OutputPath)/swig/share" -Recurse -ErrorAction 'SilentlyContinue'
-    Remove-Item "$($ConfigData.OutputPath)/swig/bin" -Recurse -ErrorAction 'SilentlyContinue'
+    Remove-Item "$($ConfigData.OutputPath)/swig" -Recurse -ErrorAction SilentlyContinue
```

Comment:
Is there really nothing left in `$($ConfigData.OutputPath)/swig` to keep?

Reply:
Not that I'm aware of - anything swig needs is its binary and the language-specific modules which sit in the `share` subdirectory.

Afaik Swig also just looks for a `Lib` directory adjacent to the binary exclusively on Windows, because there is no POSIX directory standard - more recent versions might _also_ support that, but it was the search location with the highest priority.